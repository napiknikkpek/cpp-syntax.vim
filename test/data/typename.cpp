typename T;
typename T::type;
typename T::type{};
typename T::type a;
