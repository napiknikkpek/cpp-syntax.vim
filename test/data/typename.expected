aaaaaaaa.t.
aaaaaaaa.t..tttt.
aaaaaaaa.t..tttt...
aaaaaaaa.t..tttt.x.
